packages/@aws-cdk/aws-certificatemanager/test/integ.dns-validated-certificate.ts

```diff
@@ -36,7 +36,6 @@ const hostedZone = PublicHostedZone.fromHostedZoneAttributes(stack, 'HostedZone'
 const cert = new DnsValidatedCertificate(stack, 'Certificate', {
   domainName,
   hostedZone,
-  validation: CertificateValidation.fromDns(hostedZone),
```

Comment:
This certainly feels like it should be redundant (as the `fromDns` is covered by virtue of being a `DnsValidateCertificate`, and the `hostedZone` is passed as its own prop). And couldn't see it getting used in `dns-validated-certificate.ts` anywhere.